panel/src/components/View/Menu.vue

```diff
@@ -31,6 +31,16 @@
 					class="k-panel-menu-button"
 				/>
 			</menu>
+			<menu v-if="$panel.license === false">
```

Comment:
Probably a bit of an edge case, but in Safari, the optional status bar (disabled by default and if enabled, only displayed when hovering a link for example) is displayed right where the activation menu and bubble is. Don't know if and how we would avoid that as we would need a different/higher position.

<img width="590" alt="Bildschirmfoto 2023-11-04 um 16 10 15" src="https://github.com/getkirby/kirby/assets/1595007/89bef98e-72b3-44f0-bba6-5767ef2b62bd">

Reply:
The only solution for this would be a bottom margin for the last item in the menu, but I think it would look pretty weird.

Reply:
Yeah, it would need 1.5rem of margin. Or padding for the menu body. I had never really noticed that before, but with the colorful menu button and bubble as the last item, it is much more noticeable than with a text-and-icon-only menu button and feels very weird.

Could we maybe reorder the menu again so it's not last?

Reply:
The beauty of having it as the last item, is the alignment with the notification on the right. This cleans up the bottom part nicely. If we move it up again, it will stick out a lot more annoyingly than it does now and I don't like the idea of that. I would keep it like this tbh.

Reply:
I think it's ok, it's only for those with that floating status bar and only when hovering a link partially overlaid

Reply:
It's OK for me, if just slightly annoying. But I guess I and others can get used to it. Please test it though, in the real world it is more weird than in the screenshot.

Reply:
https://github.com/getkirby/kirby/assets/3788865/c1215ebb-a916-432e-8a2c-599838fff535

What exactly do you mean by flicker?

Actually, I found it in real even less a problem than from the screenshot. It's the same in Chrome btw. Only that the status bar is black there, not the glass effect.

Reply:
Ah yes, that's identical as in Chrome (indeed the color is just dark mode).

What I mean with flickering is that the status bar is not there all the time, but keeps appearing and disappearing if you hover and no longer hover something. So in the corner of the eye, it's like a blinking pink box. If it was constantly covered, it would look worse from a design point of view, but it would at least be better from a UX point of view.

Reply:
But is that experience so much different without the activation bubble? What you describe feels to me like the general issue (for some people, for others not) with that status bar not having a fixed place but coming in and out, in and out. To me visually it's not much differently to when it comes in over the white background within that bubble.

Reply:
The black status bar over the grey menu, sure. But for me it was new because the light-mode status bar is so subtle that I never noticed that it covered the last menu item. With the pink button, it's a different story.

Reply:
So yes, I think my brain just needs to adapt to this and then I'll not notice it. It's going to be the same for our users though, so some kind of solution would help a lot. But if we can't find one that doesn't create other design problems, then it's fine.